test/UniversalRewardsDistributorTest.sol

```diff
@@ -220,14 +220,14 @@ contract UniversalRewardsDistributorTest is Test {
         vm.warp(block.timestamp + timeElapsed);
 
         vm.prank(randomCaller);
-        vm.expectRevert(bytes(ErrorsLib.TIMELOCK_NOT_EXPIRED));
+        vm.expectRevert(bytes(ErrorsLib.TIMELOCK_NOT_ELAPSED));
         distributionWithTimeLock.acceptRoot();
     }
 
     function testAcceptRootShouldRevertIfNoPendingRoot(address randomCaller) public {
         vm.prank(randomCaller);
         vm.expectRevert(bytes(ErrorsLib.NO_PENDING_ROOT));
-        distributionWithTimeLock.acceptRoot();
+        distributionWithoutTimeLock.acceptRoot();
```

Comment:
This is a stronger test because without timelock it reverts less often (so now even if we omit the argument of `expectRevert` it will catch errors where we forget the `require`)